go.mod

```diff
@@ -4,6 +4,9 @@ go 1.12
 
 require (
 	github.com/dgrijalva/jwt-go v3.2.0+incompatible
+	github.com/digitalocean/godo v1.35.1
+	github.com/tidwall/gjson v1.6.0
+	github.com/tidwall/pretty v1.0.1 // indirect
```

Comment:
what does `// indirect` mean here?

Reply:
I wanted to install gjson and im guessing that the indirect packages are dependencies that gjson requires